bundles/edu.kit.kastel.sdq.eclipse.grading.core/src/edu/kit/kastel/sdq/eclipse/grading/core/ArtemisController.java

```diff
@@ -212,35 +286,59 @@ public IExercise getExerciseFromCourses(List<ICourse> courses, int courseID, int
 	public List<IExercise> getExercisesFromExam(final String examTitle) {
 		return this.getExercisesFromExam(examTitle, this.getCourses());
 	}
+	
+	@Override
+	public Date getCurrentDate() {
+		try {
+			return this.artemisClient.getTime();
+		} catch (ArtemisClientException e) {
+			return new Date();
+		}
+	}
 
 	private List<IExercise> getExercisesFromExam(final String examTitle, List<ICourse> courses) {
-		IExam foundExam = null;

```

Comment:
Do we use this method for grading as well? If so the code might break correction mode?